### DISCUSSION TITLE
Ikea bulb unsupported

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#13874] [Devices]
Hi.
I have Ikea bulb(LED1924G9).
When I pair it with Z2M, it says not supported.
How can I resolve this?
<img width="449" alt="image" src="https://user-images.githubusercontent.com/12578699/188484649-2168da7e-a4fc-454e-bd38-8b2428e4a144.png">

**Comment by Koenkk** [comment]
This device is supported in z2m 1.27.2

**Reply by ghost** [reply]
But version of my Z2M is already 1.27.2.
<img width="299" alt="image" src="https://user-images.githubusercontent.com/12578699/188618474-dff24e41-e174-4ab2-a71c-b1339b6e1b94.png">

**Reply by Koenkk** [reply]
How are you running z2m?

**Reply by ghost** [reply]
I run z2m as home assistant addon.

**Reply by Koenkk** [reply]
Ah wait, it the model is slightly different. Added it now.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by cautje** [reply]
Hi Koen,

Here also a IKEA bulbnot supported.

Support status
Not supported

IEEE Address
0x000b57fffee80585
Network address
0xCDF7


<img width="641" alt="image" src="https://user-images.githubusercontent.com/19371264/190990982-9529a4c6-5d72-4b56-b1c7-5dc24667cf9d.png">


I ame already on the EDGE version.

Please let me know if i can help.

THX!

**Reply by cautje** [reply]
on zigbee2MQTT

**Reply by cautje** [reply]
<img width="536" alt="image" src="https://user-images.githubusercontent.com/19371264/191031905-0df2cfe7-c419-4d59-9c40-4f0927034002.png">

**Reply by cautje** [reply]
@Koenkk 
Hi Koen, i ahe added the overview of the bulb.
THX!!!

**Comment by scoobydrvr** [comment]
I have this same problem with an IKEA E1812 shortcut button. I'm running z2m on a Raspberry Pi with an ezsp-based adapter. The log page says the device is failing the interview step and is not supported. When I moved to the dev branch, the button fails to include into the network altogether.

**Reply by scoobydrvr** [reply]
I replaced my adapter with Tube's CC2652P2 based Zigbee to Ethernet/USB Serial Coordinator and now the buttons work fine. EZSP seems to have been the problem.
